sass/components/_avatar.scss

```diff
@@ -0,0 +1,34 @@
+.avatar {
+  object-fit: cover;
+  width: 40px;
+  height: 40px;
```

Comment:
Hi Vinit,
The `width` and `height` in the `avatar` class seem redundant as you are mentioning the same for `md` class.